tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -328,6 +338,7 @@ trait WebActionsApiTests extends ControllerTestCommon with BeforeAndAfterEach wi
 
         it should s"not match invalid routes (auth? ${creds.isDefined})" in {
             implicit val tid = transid()
+            customOptions = true
```

Comment:
should options be pulled out and run in its own tests (do we need more than two?)